.gitignore

```diff
@@ -4,3 +4,4 @@ build
 composer.lock
 phpunit.xml
 vendor
+.idea
```

Comment:
Please make sure your editor adds a trailing newline character at the end of files.